callHealthCheck.ts

```diff
@@ -1,32 +1,30 @@
 import axios from 'axios';
 import https from 'https';
 
-const {
-    FULL_URL,
-    API_KEY
-} = process.env;
+const { FULL_URL, API_KEY } = process.env;
 
-exports.callHealthCheck = async function(){
-        console.log(`Timer Function Called`)
-        var url =  FULL_URL;
-        const agent = new https.Agent({
-            rejectUnauthorized: false
-        });
-        var options = {
-            httpsAgent: agent,
-            headers: {
-                Authorization: 'Bearer ' +  API_KEY
-            }
-        };
-        try {
-            console.log(`Calling ${FULL_URL}`)
-            const response = await axios.get(url, options);
-            console.log(response.data)
-            return response.data;
-        } catch (err) {
-            throw console.error(`Error retrieving VIP data from Fortigate: ${url}. Error : ${err} `);
+exports.callHealthCheck = async function(event, context, callback): Promise<void> {
+    console.log('Timer Function Called');
+    var url = FULL_URL;
+    const agent = new https.Agent({
+        rejectUnauthorized: false
+    });
+    var options = {
+        httpsAgent: agent,
+        headers: {
+            Authorization: 'Bearer ' + API_KEY
         }
+    };
+    try {
+        console.log(`Calling ${FULL_URL}`);
+        await axios.get(url, options);
+    } catch (err) {
+        console.error(`Error in CallHealthCheck Function. Error : ${err} `);
+        callback(err);
     }
+    // Terminate function.
+    callback(null, 'CallHealthCheck Terminated.');
+};
 if (module === require.main) {
     exports.callHeallthCheck(console.log);
```

Comment:
Still hea**ll**thCheck, which is obviously wrong right?

Reply:
This section is more for local testing, but it calls the health check function if that is what you are asking.